### PR TITLE
4 4 branch

### PR DESCRIPTION
**Pull request by mcleodaj**

### CODE REVIEW

**Comment by mcleodaj** [comment]
Code review? @bontgoy

**Comment by Bontgoy** [comment]
If requesting feedback make sure to wait for my reply before merging the pull request

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+def foodMethod (food)
```
Comment (Bontgoy) [inline ruby/vampires.rb:1]: no space between method name and param.

also ruby is snake case `food_method` not camel case `foodMethod`

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  puts "Do you have any allergies? (type 1 at a time until finished, type 'done' when finished or if no allergies)"
+  while answer!="done"
+    answer=gets.chomp
+    if answer=="sunshine"
+      result="Probably a vampire"
+      break
+    end
+  end
+
+  yearDiff=2016-age
```
Comment (Bontgoy) [inline ruby/vampires.rb:44]: with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.now.year`

File: ruby/vampires.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    if correctYear==true && (food || insurance)==true
+      result="Probably not a vampire"
+    end
+    if correctYear==false && (food==false || insurance==false)
+      result="Probably a vampire"
+    end
+    if correctYear==false && food==false && insurance==false
+      result="Almost certainly a vampire"
+    end
+    if name=="Drake Cula" || name=="Tu Fang"
```
Comment (Bontgoy) [inline ruby/vampires.rb:56]: I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```

**Pull request merged**